pkgs/tools/networking/reaver-wps-t6x/default.nix

```diff
@@ -1,33 +1,24 @@
-{ stdenv, fetchFromGitHub, libpcap, sqlite, pixiewps, makeWrapper }:
+{ stdenv, fetchFromGitHub, libpcap, pixiewps, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  version = "1.5.2";
+  version = "1.6.3";
   name = "reaver-wps-t6x-${version}";
-  confdir = "/var/db/${name}"; # the sqlite database is at "${confdir}/reaver/reaver.db"
 
   src = fetchFromGitHub {
     owner = "t6x";
     repo = "reaver-wps-fork-t6x";
     rev = "v${version}";
-    sha256 = "0zhlms89ncqz1f1hc22yw9x1s837yv76f1zcjizhgn5h7vp17j4b";
+    sha256 = "1bccwp67q1q0h5m38gqxn9imq5rb75jbmv7fjr2n38v10jcga2pb";
   };
 
   nativeBuildInputs = [ makeWrapper ];
-  buildInputs = [ libpcap sqlite pixiewps ];
+  buildInputs = [ libpcap pixiewps ];
 
-  setSourceRoot = ''
-    sourceRoot=$(echo */src)
-  '';
-
-  configureFlags = "--sysconfdir=${confdir}";
+  sourceRoot = "reaver-wps-fork-t6x-v${version}-src/src";
 
```

Comment:
Why did you change the `sourceRoot` here? As is the package won't build because of this change.